include/bh_platform.h

```diff
@@ -111,6 +111,18 @@ typedef int bh_socket_t;
 #  define NULL ((void*) 0)
 #endif
 
+/**
+ * Return the offset of the given field in the given type.
+ *
+ * @param Type the type containing the filed
+ * @param field the field in the type
+ *
+ * @return the offset of field in Type
+ */
+#ifndef offsetof
+#define offsetof(Type, field) ((size_t)(&((Type *)0)->field))
```

Comment:
This is already defined [here](https://github.com/unikraft/unikraft/blob/e467cdf69ece968187523a7cd6ba87fa9ed6f34f/include/uk/essentials.h#L292), so you can include the header.

Reply:
Addressed in new commit.